svelte/lib/src/components/progressbar/Progressbar.svelte

```diff
@@ -1,37 +1,33 @@
-<script lang="ts" context="module">
-	import type {ProgressbarProps as Props, ProgressbarSlots as Slots} from '@agnos-ui/svelte-headless/components/progressbar';
-	import {createProgressbar} from '@agnos-ui/svelte-headless/components/progressbar';
+<script lang="ts">
+	import {createProgressbar, type ProgressbarProps as Props, type ProgressbarSlots as Slots} from '@agnos-ui/svelte-headless/components/progressbar';
 	import {Slot} from '@agnos-ui/svelte-headless/slot';
 	import {callWidgetFactory} from '@agnos-ui/svelte-headless/config';
 	import ProgressbarDefaultStructure from './ProgressbarDefaultStructure.svelte';
-	const defaultConfig: Partial<Props> = {
-		slotStructure: ProgressbarDefaultStructure,
-	};
-</script>
 
-<script lang="ts">
-	// cf https://github.com/ota-meshi/eslint-plugin-svelte/issues/348
-	type $$Props = Partial<Props>; // eslint-disable-line @typescript-eslint/no-unused-vars
-	type $$Slots = Slots; // eslint-disable-line @typescript-eslint/no-unused-vars
-	export const widget = callWidgetFactory({factory: createProgressbar, widgetName: 'progressbar', $$slots, $$props, defaultConfig, events: {}});
+	type $$Props = Partial<Props>;
+	type $$Slots = Slots;
+
+	const widget = callWidgetFactory({
+		factory: createProgressbar,
+		widgetName: 'progressbar',
+		$$slots,
+		$$props,
+		defaultConfig: {
+			slotStructure: ProgressbarDefaultStructure,
+		},
+		events: {},
+	});
 	const {
-		stores: {slotStructure$, ariaLabel$, value$, min$, max$, ariaValueText$},
+		stores: {slotStructure$},
 		state$,
 	} = widget;
+	export const api = widget.api;
 
 	$: widget.patchChangedProps($$props);
-
 	$: slotContext = {widget, state: $state$};
 </script>
 
-<div
-	role="progressbar"
-	aria-label={$ariaLabel$ || undefined}
-	aria-valuenow={$value$}
-	aria-valuemin={$min$}
-	aria-valuemax={$max$}
-	aria-valuetext={$ariaValueText$}
->
+<div use:widget.directives.ariaDirective>
```

Comment:
It's only used once, but I wonder if it's not better, for clarity, to have this directive declared as the others, in the destructuration of `widget`